packages/idx/src/property/search.component.ts

```diff
@@ -381,6 +386,10 @@ Stratus.Components.IdxPropertySearch = {
         }
 
         $scope.$watch('options.query.where.ListingType', () => {
+            // on load this isn't defined yet
+            if (typeof $scope.options === 'undefined' || typeof $scope.options.query === 'undefined' ) {
```

Comment:
if `$scope.options` isn't defined the watcher for `$scope.options.query.where.ListingType` couldn't have been able to even kick off

Reply:
Well, I don't know why, but this line was causing fatal errors until I added this check for `$scope.options === 'undefined'`. Once I added this, the errors went away. So somehow this was kicking off.